code/__DEFINES/misc_defines.dm

```diff
@@ -138,6 +138,9 @@
 #define MIN_SUPPLIED_LAW_NUMBER 15
 #define MAX_SUPPLIED_LAW_NUMBER 50
 
+/// Grabs the area of a supplied object. Passing an area in to this will result in an error
+#define get_area(T) ((get_step(T, 0).loc))
```

Comment:
If you want to know why this doesn't just call the get_turf macro, it's because tet operations REALLY don't like that